.gitignore

```diff
@@ -2,3 +2,5 @@
 *~
 *#
 .#*
+build
```

Comment:
It seems too broad. 
Could you please explain why you think it is needed?

Reply:
Forgot the trailing slash, but here's a pretty standard `.gitignore` for distribution: https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Python.gitignore#L9